webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/reports/NodeComplianceExpiration.scala

```diff
@@ -0,0 +1,33 @@
+package com.normation.rudder.domain.reports
+
```

Comment:
this new class is the holder for the configuration setting for a node (based on what the property holds)

Reply:
OK, does it need the license header ?